lib/helper/Appium.js

```diff
@@ -1350,7 +1350,16 @@ class Appium extends Webdriver {
     if (this.isWeb) return super.grabValueFrom(locator);
     return super.grabValueFrom(parseLocator.call(this, locator));
   }
-
+  
+  /**
+   * {{> scrollIntoView }}
```

Comment:
may you add the documentation for this?

Reply:
@PeterNgTr added docu here: docs/webapi/scrollIntoView.mustache